src/views/CommunityModeling.vue

```diff
@@ -296,6 +406,7 @@ export default Vue.extend({
         .post(`${settings.apis.simulations}/community/simulate`, payload)
         .then(response => {
           this.communityData = response.data;
+          this.renderCrossfeeding();
```

Comment:
It could be a good idea to dispose the previous chart before creating the new one, I'm getting these warnings when re-simulating:

![image](https://user-images.githubusercontent.com/64745/74137397-0406a500-4bf0-11ea-8fa3-c706a38d9797.png)